aerogear-android-push/src/main/java/org/jboss/aerogear/android/unifiedpush/metrics/MetricsMessage.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * JBoss, Home of Professional Open Source
+ * Copyright Red Hat, Inc., and individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * 	http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.aerogear.android.unifiedpush.metrics;
+
+public interface MetricsMessage {
+
+    /**
+     * A messageId. Some id that the receiver of this message can relate to.
+     */
+    public String getMessageId();
```

Comment:
let's not have the `public` here. The other new interface (`MetricsSender.java`) also does not explicitly add the `public`